pynq_composable/composable.py

```diff
@@ -81,6 +81,32 @@ def _get_ip_name_by_vlnv(description: dict, vlnv: str) -> str:
             return k
     return None
 
+def _streamline_pipeline(pipe: list) -> list:
+    """
+    Flattens List
```

Comment:
Even though this is an internal function a good practice is to write good documentation. 
This function is key in the internal logic, so please spend some time documenting it well